docs/getting-started.md

```diff
@@ -29,20 +28,21 @@ newgrp docker
 
 2) **Set up DNS:**
 
-    Hyku makes heavy use of domain names to determine which tenant to serve. On MacOS/Linux, it is recommended to use [Dory](https://github.com/FreedomBen/dory) to handle the necessary DNS changes.
+    Hyku makes heavy use of domain names to determine which tenant to serve. On MacOS/Linux, it is recommended to use [Stack Car](https://github.com/notch8/stack_car) to handle the necessary SSL certs and proxy setup.
 
-    #### Dory Installation
+    #### Stack Car Installation
 
     ```bash
-    gem install dory
-    dory up
+    gem install stack_car
+    sc proxy cert # Only need this once per stack_car version
```

Comment:
I got this message when I ran this command. Where's the password? 

```ruby
Enter the password found https://github.com/Upinel/localhost.direct?tab=readme-ov-file#a-non-public-ca-certificate-if-you-have-admin-right-on-your-development-environment-you-can-use-the-following-10-years-long-pre-generated-self-signed-certificate
```

Reply:
nevermind, i followed the link to where it said the password was. Could we add that to the readme?

Reply:
added more info to getting-started.md